ToolBox/BrainDataConverter/BrainDataConverter.cs

```diff
@@ -0,0 +1,246 @@
+                    );
+                }
+            }
+            catch (Exception e)
+            {
+                Log.Error(e, $"BrainDataConverter.Convert(): Failed to process five day ranking data: {rankingFiveSourceFile.FullName}");
+                success = false;
+            }
+            try
+            {
+                using (var rankingStream = rankingTenSourceFile.OpenRead())
+                {
+                    WriteToFile(
+                        Process<BrainDataRankingsTenDay>(date, rankingStream),
+                        rankingTenFinalDirectory
+                    );
+                }
+            }
+            catch (Exception e)
+            {
+                Log.Error(e, $"BrainDataConverter.Convert(): Failed to process ten day ranking data: {rankingTenSourceFile.FullName}");
+                success = false;
+            }
+
+            return success;
+        }
+
+        /// <summary>
+        /// Skips the header row of raw Brain Data data
+        /// </summary>
+        /// <param name="line"></param>
+        /// <returns></returns>
+        public override bool ShouldSkipLine<T>(string line)
+        {
+            return line.StartsWith("DATE");
+        }
+
+        /// <summary>
+        /// Converts formatted data to an instance of <typeparamref name="T"/>
+        /// </summary>
+        /// <typeparam name="T">BaseData derived type</typeparam>
+        /// <param name="ticker">Ticker of the data we're processing</param>
+        /// <param name="line">Line of formatted data. This should come from the Data Folder</param>
+        /// <returns>Instance of T</returns>
+        public override T GetDataInstance<T>(string ticker, string line)
+        {
+            var mockSubscription = new SubscriptionDataConfig(
+                typeof(T),
+                Symbol.Create(ticker, SecurityType.Base, Market.USA),
+                Resolution.Daily,
+                TimeZones.Utc,
+                TimeZones.Utc,
+                false,
+                false,
+                false);
+
+            var instance = new T();
+            return (T)instance.Reader(mockSubscription, line, DateTime.MinValue, false);
+        }
+
+        /// <summary>
+        /// Converts raw data to instance of <typeparamref name="T"/>
+        /// </summary>
+        /// <typeparam name="T">BaseData derived type</typeparam>
+        /// <param name="line">Line of unformatted/raw data</param>
+        /// <returns>Instance of T</returns>
+        public override T GetDataInstanceFromRaw<T>(string line)
+        {
+            var csv = line.ToCsv();
+
+            if (typeof(T) == typeof(BrainDataSentimentWeekly))
+            {
+                return (T)(BaseData)new BrainDataSentimentWeekly
+                {
+                    Time = DateTime.ParseExact(csv[0], "yyyy-MM-dd", CultureInfo.InvariantCulture),
+                    Symbol = Symbol.Create(csv[1], SecurityType.Base, Market.USA),
```

Comment:
`Symbol.Create` is not always the best alternative. under the hood, it calls `GenerateEquity`
https://github.com/QuantConnect/Lean/blob/9306f262c19782636686d0334818213b130f6368/Common/SecurityIdentifier.cs#L351

Meaning that for each line it will try to load a `MapFileProvider`.